cpp/src/io/utilities/parsing_utils.cuh

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <vector>
+
+#include "cudf.h"
+
+using ll_uint_t = unsigned long long int;
+
+gdf_error countAllFromSet(const char *h_data, size_t h_size, std::vector<char> keys, 
+	gdf_size_type &rec_cnt);
+
+gdf_error findAllFromSet(const char *h_data, size_t h_size, std::vector<char> keys, ll_uint_t result_offset,
+	ll_uint_t *recStart);
```

Comment:
Missing newline at end of file. Some IDEs and tools require a line ending at the end